deploy-v2.js

```diff
@@ -0,0 +1,162 @@
+const {
+  Web3Connection,
+  ERC20,
+  BountyToken,
+  Network_v2,
+  NetworkFactoryV2,
+  toSmartContractDecimals,
+  Network_Registry
+} = require("@taikai/dappkit");
+const { nativeZeroAddress } = require("@taikai/dappkit/dist/src/utils/constants");
+const fs = require("fs");
+
+const cap = toSmartContractDecimals("50000000", 18);
+
+const connection = new Web3Connection({
+  web3Host: "http://127.0.0.1:8545",
+  privateKey:
+    "0x40a674d5327e433ccb291408db12e890a806bfa49916e2fd948770b246d6a871",
```

Comment:
hardcoded pvtkeys can't be a thing -- lets all hope this is a demo private key and _not your own_